search-sentiments-map/routes/trends.js

```diff
@@ -36,6 +41,42 @@ router.get('/', (req, res) => {
   });
 });
 
+/** Renders the popularity data of the country given by the request parameter. */
+router.post('/', jsonParser, (req, res) => {
+  googleTrends.interestOverTime({
+    keyword: req.body.topic,
+    startTime: new Date(Date.now() - (DATA_FROM_8_DAYS_AGO_MS)),
+    geo: req.body.code,
+  }).then(data => {
+    res.setHeader('Content-Type', 'application/json');
+    res.send(data);
+    })
+  .catch(err => {
```

Comment:
```suggestion
  }).catch(err => {
```

Reply:
There's a correct amount of bracket pairs already?

Reply:
yes it's the indent